frontend/src/pages/signin-page.tsx

```diff
@@ -90,13 +95,20 @@ function signin() {
             )}
           </div>
 
-          <div className="mb-4">
+          <div className="mb-4 relative">
             <input
               {...register('password')}
-              type="password"
+              type={passwordVisible ? 'text' : 'password'}
               placeholder="Password"
               className="w-full rounded-lg bg-zinc-100 p-3 font-normal placeholder:text-sm dark:bg-dark-field dark:text-dark-textInField"
             />
+            <button
+              type="button"
+              onClick={() => setPasswordVisible(!passwordVisible)}
+              className="absolute inset-y-0 right-0 flex items-center pr-3 text-sm leading-5"
+            >
```

Comment:
Kindly make sure if this works in the mobile screen as well, can you add the video recorded for the mobile screen as well

Reply:
sure @krishnaacharyaa